Branch-SDK/src/io/branch/indexing/ContentDiscoverer.java

```diff
@@ -190,16 +198,11 @@ private void discoverFilteredViewContents(JSONArray viewIDArray, JSONArray conte
 
     private void discoverViewContents(ViewGroup viewGroup, JSONArray contentDataArray, JSONArray contentKeysArray, Resources res, boolean isClearText) {
         for (int i = 0; i < viewGroup.getChildCount(); i++) {
-            
             View childView = viewGroup.getChildAt(i);
-            if ((childView.getVisibility() == View.VISIBLE) && (childView instanceof ViewGroup)) {
+            if (childView.getVisibility() == View.VISIBLE) if (childView instanceof ViewGroup) {
                 discoverViewContents((ViewGroup) childView, contentDataArray, contentKeysArray, res, isClearText);
             } else {
-                String viewName = String.valueOf(childView.getId());
```

Comment:
No sure how this changes came back. This will crash  in some case we need  the try catch as it is  there